### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by krate27**
By mistake I made one feature of products page in this branch, please review this with home page only.

### CODE REVIEW

**Review by Mohitkumar33 — APPROVED** [review]

File: src/Components/Filters/filter.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.filter-class {
+    display: flex;
+    justify-content: space-between;
+    margin: 8rem 2rem 1rem;
```
Comment (samyakshah3008) [inline src/Components/Filters/filter.css:4]: Good use of shorthand property ✅

File: src/Components/Filters/filter.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+.basic-image {
+
+    width: 100%;
+    object-fit: contain;
+}
+.main-body{
+display: flex;
+flex-direction: column;
+}
```
Comment (samyakshah3008) [inline src/Components/Filters/filter.css:45]: Can make it clean with format document(refactor later on)

File: src/Components/ProductListing/productListing.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+              style={{width:'20rem'}}
+            />
+          </picture>
+          <main className="main-body">
+            <h3 className="card-title"> {title} </h3>
+            <p>( {catchName} )</p>
+            {/* <p>
+              Full Sleeve Graphic Print Men Sweat-shirt Punisher based theme for
+              men....
+            </p> */}
```
Comment (samyakshah3008) [inline src/Components/ProductListing/productListing.jsx:32]: Can remove this commented code if not necessary

**Review by samyakshah3008 — APPROVED** [review]
Nice and clean UI 
Loved your folder structure 
Given some minor suggestion, you can refactor later on 

Rest is all good for approval ✅

File: src/custom-hooks/useFetch.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            console.log("fetch aborted");
+          } else {
+            setLoader(false);
+            setError(error);
+          }
+        }
+      })();
+    
+
+    return () => abortCont.abort();
```
Comment (Varun-28) [inline src/custom-hooks/useFetch.jsx:30]: - well-structured code
- great use of custom hooks
- good logic building

**Pull request merged**